guide/payments/transactions.md

```diff
@@ -32,18 +32,67 @@ transactions, it's important for designers to understand these differences when
+This page is meant to help you understand how bitcoin transactions get created and processed by the Bitcoin and Lightning networks. Bitcoin is a peer-to-peer push-payment system. This means that you can send, or push, bitcoin at any time, without passing through a trusted third party.
 
-This is radically different from the traditional financial system, where it is often possible for others to *pull* and withdraw money from your account (utility companies, financial institutions, merchants, etc.). When you make a payment, it will pass through systems that might delay, control or block the payment.
+This is radically different from the traditional financial system, where it is often possible for others to pull and withdraw money from your account (utility companies, financial institutions, merchants, etc.). When you make a payment, it will pass through systems that might delay, control or block the payment.
 
-## Transaction lifecycle
+## Comparing Bitcoin and Lightning
 
-Let's lay out the entire payment process.
+| Bitcoin          | Lightning |
+|-------------|------------------|
+| Users send *[transactions]({{'/guide/glossary/#transaction' | relative_url}})*. | Users send *[payments]({{'/guide/glossary/#payment' | relative_url}})*.  |
+| Transactions are settled on average every 10 minutes. | Payments are settled almost instantly.  |
+| A valid transaction will always be mined, but may be delayed if the fee is too low. | Successful payment depends on Lightning network liquidity and finding a path to the recipient. |
+| Bitcoin is typically better for larger amounts of funds and less frequent transactions. | Lightning is typically better for smaller amounts of funds and more frequent payments. |
+
+For a more detailed look, see the [Technology Primer]({{'/guide/getting-started/technology-primer/' | relative_url}}).
+
+## Lightning payment lifecycle
+
+For most payments, the Lightning network is the best option, as it will enable the recipient to receive their bitcoin quicker and the sender to pay less in fees.
+
+{% include picture.html
+image = "/assets/images/guide/payments/transactions/payment-lifecycle.svg"
+mobile = "/assets/images/guide/payments/transactions/payment-lifecycle-mobile.svg"
+alt-text = "Flow chart of the process of a Lightning payment from invoice creation through successful payment"
+width = 1300
+height = 240
+layout = "full-width"
+%}
+
+### 1. Create invoice
+
+To make a payment, the sender requires an invoice created by the recipient. The recipient's wallet application creates a preimage, or payment secret, which is then used to create an invoice for one-time use. The recipient can share the invoice with the sender as a QR code, over NFC, in plain text, or as a payment link.
```

Comment:
Is invoice creation always a step? How about with LN-URL or keysend?

Reply:
WIth LNURL, invoice creation is a step; however, the actual invoice is generated by a remote server.

With keysend, there is no invoice. Personally, I'm not interested in talking much about keysend. I'm not convinced it's a technique that should be promoted, and I think other solutions will become more widely used.

What do you think? Should I add some more nuance to this section? e.g. "happens like this most of the time, but not all the time".

Reply:
I was mostly curious how you think about these things. Personally for me things like LNURL-Pay, Bolt12 and keysend don't really feel like invoice creation,  at least from the user-perspective.

Mostly, just want to make sure we're consistent on how we talk about things, especially because of the things in #589 

An easy work-around would probably be to add, `most of the times invoice created by the recepient`

Reply:
I added some details about this in 08a4ce0.

Reply:
Fantastic, thank you for addressing that one.